cdf-core/resource/messages_ja.properties

```diff
@@ -1,6 +1,6 @@
-CdfContentGenerator.ERROR_0001_NO_OUTPUT_HANDLER=\u51fa\u529b\u30cf\u30f3\u30c9\u30e9\u30fc\u304c\u3042\u308a\u307e\u305b\u3093\u3002
-CdfContentGenerator.ERROR_0002_NO_CONTENT_ITEM=\u30b3\u30f3\u30c6\u30f3\u30c4\u9805\u76ee\u304c\u3042\u308a\u307e\u305b\u3093\u3002
-CdfContentGenerator.ERROR_0003_NO_OUTPUT_STREAM=\u51fa\u529b\u30b9\u30c8\u30ea\u30fc\u30e0\u304c\u3042\u308a\u307e\u305b\u3093\u3002
-CdfContentGenerator.ERROR_0004_NO_REQUEST_PARAMS=\u30ea\u30af\u30a8\u30b9\u30c8 \u30d1\u30e9\u30e1\u30fc\u30bf\u30fc\u304c\u3042\u308a\u307e\u305b\u3093\u3002
-CdfContentGenerator.ERROR_0005_BAD_TEMPLATE_OBJECT=\u4e0d\u6b63\u306a\u30c6\u30f3\u30d7\u30ec\u30fc\u30c8 \u30aa\u30d6\u30b8\u30a7\u30af\u30c8\u3067\u3059\u3002
-CdfFileInfoGenerator.ERROR_0001_PARSING_XCDF=XCDF \u30d5\u30a1\u30a4\u30eb\u306e\u89e3\u6790\u4e2d\u306b\u30a8\u30e9\u30fc\u304c\u767a\u751f\u3057\u307e\u3057\u305f\u3002
+CdfContentGenerator.ERROR_0001_NO_OUTPUT_HANDLER=No Output Handler
```

Comment:
Is this correct ? It looks as if the english message is in the japanese properties file.